logspout.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+
+services:
+  logspout:
+    build: logspout/
+    volumes:
+      # security concern:
+      # https://raesene.github.io/blog/2016/03/06/The-Dangers-Of-Docker.sock/
+      # http://stackoverflow.com/questions/40844197
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    environment:
+      ROUTE_URIS: "logstash+tcp://logstash:5000"
```

Comment:
What's the advantage of using this over `github.com/gliderlabs/logspout/transports/tcp`?